docs/source/notes/amp_examples.rst

```diff
@@ -380,20 +380,20 @@ Functions that need a particular ``dtype``
 ------------------------------------------
 
 Consider a custom function that requires ``torch.float32`` inputs.
-Apply :func:`custom_fwd(cast_inputs=torch.float32)<custom_fwd>` to ``forward``
-and :func:`custom_bwd<custom_bwd>` (with no arguments) to ``backward``.
+Apply :func:`custom_fwd(device_type='cuda', cast_inputs=torch.float32)<custom_fwd>` to ``forward``
+and :func:`custom_bwd(device_type='cuda')<custom_bwd>` to ``backward``.
```

Comment:
Should a note below be adjusted for XPU? "If ``forward`` runs in an autocast-enabled region, the decorators cast **floating-point CUDA Tensor** inputs to ``float32``"

Reply:
The intention of this pr is to make this file device-agnostic. Not planning to highlight a specific device.

Reply:
> The intention of this pr is to make this file device-agnostic.

Yes, and right now this paragraph talks about cuda.

Reply:
@jingxu10, @svekars : I would appreciate if this comment would have been answered before the merge. I still believe this part of the documentation might need an update and that's why I keep -1 on this PR.

Reply:
Could you take a review on the edit?

Reply:
reads good to me now.